test/extended/image_ecosystem/plugin.go

```diff
@@ -378,6 +378,19 @@ func getAdminPassword(oc *exutil.CLI) string {
 	return "password"
 }
 
+// Finds the pod running Jenkins
+func findJenkinsPod(oc *exutil.CLI) *kapi.Pod {
```

Comment:
There is some similar logic in test/extended/util/framework.go that searches for pods (DumpDeploymentLogs for example).  Perhaps we should somehow refactor this method with those, making this one more generic and take in args for the prefix / contains calls (maybe even make the phase a param?) and get some reuse.